src/caches/basic_caches.jl

```diff
@@ -10,18 +10,16 @@ mutable struct CompositeCache{T,F} <: OrdinaryDiffEqCache
   current::Int
 end
 
-function alg_cache(alg::CompositeAlgorithm,u,rate_prototype,uEltypeNoUnits,uBottomEltypeNoUnits,tTypeNoUnits,uprev,uprev2,f,t,dt,reltol,p,calck,::Val{false})
-  caches = map(alg.algs) do x
```

Comment:
Same here, is it sufficient to use a `let` block?

Reply:
No it wasn't.